data/json/items/gun/40mm.json

```diff
@@ -0,0 +1,29 @@
+[
+  {
+    "id": "mark19",
+    "copy-from": "gun_base",
```

Comment:
You may wish to consider `launcher_base` in `classes` which includes a reasonably standard set of gunmods. In particular why are we allowing `muzzle` and `bore` mods for this item?